client/my-sites/google-my-business/stats/chart.js

```diff
@@ -118,41 +120,59 @@ class GoogleMyBusinessStatsChart extends Component {
 		);
 	}
 
-	onIntervalChange = event =>
+	changeInterval = event =>
```

Comment:
we usually prefix those handlers with `handle` to make it clear that it's a handler:
```
handleChangeInterval = event => ...
``
`